examples/softbody_usage/src/main.cpp

```
@@ -0,0 +1,218 @@
+class GraphicExampleError : public std::runtime_error {
+  public:
+    using std::runtime_error::runtime_error;
+};
+
+int main(void)
+{
+    Engine::Core core;
+
+    core.AddPlugins<Window::Plugin, Graphic::Plugin, Input::Plugin, Physics::Plugin>();
+
+    core.RegisterSystem<RenderingPipeline::Init>([](Engine::Core &core) {
```

Cppcheck (2.19.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)